lib/blocks.php

```diff
@@ -473,7 +473,8 @@ function gutenberg_block_has_support( $block_type, $feature, $default = false )
  * @return array          Metadata for registering a block type with the supports shape updated.
  */
 function gutenberg_migrate_old_typography_shape( $metadata ) {
-	if ( isset( $metadata['supports'] ) ) {
+	// Temporarily disable migrations from core blocks until core block.json are updated.
```

Comment:
Maybe we should update this comment. Now migrations are disabled to avoid false warnings when the plugin is used with WP versions older than 5.8.